internal/batches/features.go

```diff
@@ -15,6 +15,7 @@ type FeatureFlags struct {
 	AllowWorkspaces          bool
 	BatchChanges             bool
 	AllowConditionalExec     bool
+	AllowOptionalPublished   bool
 }
 
 func (ff *FeatureFlags) SetFromVersion(version string) error {
```

Comment:
Are all of our feature flags dictated by Sourcegraph version?

Reply:
Yep. Feature flag is arguably a misnomer here (my fault); these are a way for us to know what the Sourcegraph backend that `src` is talking to supports, so we can adjust `src`'s behaviour accordingly.

Reply:
Gotcha, thanks!